app/clients/telemetry.js

```diff
@@ -0,0 +1,30 @@
+const mysql = require("mysql");
+const util = require("util");
+
+const pool = mysql.createPool({
+  connectionLimit: 10,
+  host: process.env.BUOY_TELEMETRY_MYSQL_HOST,
+  user: process.env.BUOY_TELEMETRY_MYSQL_USER,
+  password: process.env.BUOY_TELEMETRY_MYSQL_PASSWORD,
+  database: process.env.BUOY_TELEMETRY_MYSQL_DATABASE,
+});
+
+const query = util.promisify(pool.query).bind(pool);
+
+const getLatestRecord = async (buoyId, tableType) => {
+  return await query(
+    `SELECT * FROM \`${buoyId}_${tableType}\` ORDER BY TmStamp desc LIMIT 1`
+  );
+};
```

Comment:
yeah, it's not a straightforward query builder task since it's creating a table name that needs to be backtick'ed (damn hyphens).  I'll take another look and see what I can find.  The values are checked on the way into the api, so nothing can get here that isn't a known entity.